services/api.ts

```diff
@@ -0,0 +1,17 @@
+import { AxiosError } from 'axios';
+
+import apiClient from '@/libs/axios';
+
+export async function postSignUpInfo(body: object) {
+  try {
+    const response = await apiClient.post('/users', body);
+    return response.data;
```

Comment:
취향이지만 이렇게 표현해줘도 괜찮을 것 같아요!
```suggestion
    const { data } = await apiClient.post('/users', body);
    return data;
```